src/mixins/EditorMixin.js

```diff
@@ -402,15 +402,18 @@ export default {
 		 * Closes the editor and returns to normal calendar-view
 		 */
 		closeEditor() {
-			const params = Object.assign({}, this.$store.state.route.params)
-			delete params.object
-			delete params.recurrenceId
+			const params = Object.assign({}, this.$route.params)
 
 			this.$router.push({
-				name: getPrefixedRoute(this.$store.state.route.name, 'CalendarView'),
+				name: getPrefixedRoute(this.$route.name, 'CalendarView'),
 				params,
 			})
-			this.$store.commit('resetCalendarObjectInstanceObjectIdAndRecurrenceId')
+
+			this.calendarObjectInstanceStore.isNew = false
+			this.calendarObjectInstanceStore.calendarObject = null
+			this.calendarObjectInstanceStore.calendarObjectInstance = null
+			this.calendarObjectInstanceStore.existingEvent.objectId = null
+			this.calendarObjectInstanceStore.existingEvent.recurrenceId = null
```

Comment:
I don't like inlining this into a mixin/component. I'll push a commit to move it back to the store.

Inlining within a store is okay, IMO.